src/include/network/services/dns.rb

```diff
@@ -244,14 +212,13 @@ def initialize_network_services_dns(include_target)
 
       @dns_td = {
         "resolv" => {
-          "header"       => _("Hostname/DNS"),
+          "header"       => _("DHCP/DNS"),
```

Comment:
Well, it is still Hostname / DNS configuration although it has the option for setting the hostname via DHCP, so, not sure if this will make the user more confuseD about what is the Tab about.

Reply:
i want to suppress any notice of this Hostname stuff when I was unable to get permission to remove it ;-) Moreover I'd like to move some DHCP pieces from global tab here in the future

Reply:
Then do it when removed / moved as currently it will a point of confusion.

Reply:
reverted